src/python/zquantum/core/interfaces/ansatz_utils_test.py

```diff
@@ -0,0 +1,71 @@
+"""Test cases for ansatz-related utilities."""
```

Comment:
`invalidates_parametrized_circuit` is not tested though it is imported elsewhere